daemon/inertiad/auth/users.go

```diff
@@ -20,9 +20,11 @@ const (
 // userProps are properties associated with user, used
 // for database entries
 type userProps struct {
-	HashedPassword string
-	Admin          bool
-	LoginAttempts  int
+	HashedPassword  string
+	Admin           bool
+	LoginAttempts   int
+	TotpKey         string
+	TOTPBackupCodes [crypto.TotpNoBackupCodes]string
```

Comment:
Should probably standardize on one of `Totp` or `TOTP` (imo the former)

Reply:
Changed to TOTPSecret

Reply:
Changed to Totp